Yank/pipeline.py

```diff
@@ -1451,6 +1486,19 @@ def _setup_system(self, system_file_path, pack, alchemical_charge,
 
             # Solvate unit. Solvent models different than tip3p need parameter modifications.
             solvent_model = solvent['solvent_model']
+            # Check that solvent model has loaded the appropriate leap parameters.
+            # This does guarantee a failure, but is a good sign of it.
+            solvent_warning = None
```

Comment:
This seems to be unused outside the `if` so you should be able to remove this line.